tests/nobiru/test_nobiru_list.py

```diff
@@ -65,3 +65,23 @@ def test_delete_with_non_existant_entity(self):
             err.exconly()
             == "werkzeug.exceptions.NotFound: 404 Not Found: DummyRelation not found"
         )
+
+    def test_find(self):
+        fake = self.create_fake_model()
+        value_id = fake.values[0].id
+
+        with patch.object(fake.values, "find") as mock_find:
+            fake.values.find(value_id)
+
+        mock_find.assert_called_once_with(value_id)
```

Comment:
I don't think it makes sense to do patching here. Essentially this is patching the find method that is executing in the test. So none of the application logic actually runs.

I think what we want to test is that the object is returned when executed. The behavior of the find method can be defined as it returns the object for the provided id; otherwise it returns an error message when not found.
```suggestion
        value = fake.values[0]
        assert value == fake.values.find(value.id)
```